nixos/modules/services/web-apps/hitide.nix

```diff
@@ -0,0 +1,53 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.hitide;
+in
+{
+  options.services.hitide = {
+    enable = mkEnableOption "the hitide service";
+
+    package = mkOption {
+      default = pkgs.hitide;
+      defaultText = "pkgs.hitide";
+      type = types.package;
+      description = "hitide package to use.";
+    };
+
+    backendHost = mkOption {
+      default = null;
+      type = types.string;
+      description = "URL path to lotide";
+      example = "http://localhost:3333/";
+    };
+
+    port = mkOption {
+      default = 4333;
+      type = types.int;
+      description = "Port number to bind to";
+    };
+  };
+
+  config = mkIf cfg.enable {
+    systemd.services.hitide = {
+      description = "hitide Service";
+      wantedBy = [ "multi-user.target" ];
+      after = [ "networking.target" "lotide.service" ];
+      serviceConfig = {
```

Comment:
Some people might want a salad of `systemd` sandboxing options here. If you're unfamiliar and interested you could read up on it [here](https://www.ctrl.blog/entry/systemd-service-hardening.html). I'm not too picky, though.